starlette/websockets.py

```diff
@@ -62,8 +53,6 @@ async def receive(self) -> Message:
                 )
             if message_type == "websocket.disconnect":
                 self.client_state = WebSocketState.DISCONNECTED
-                if "code" not in message:
```

Comment:
yes, further reading of standard indicates that this code must be application level only.